debug.cabal

```diff
@@ -24,6 +24,9 @@ data-files:
     html/debug.html
     html/debug.js
 
+extra-source-files:
+    test/ref/*.json
```

Comment:
Generally never use wildcards in .cabal files - they go badly wrong and upload gunk to Hackage.